### ISSUE TITLE
Error on creating new account

### ISSUE THREAD

**Issue by brunosalmeida** [blockchain/service-my-wallet-v3#227]
Debian 8
Node = 6.10.3
service-my-wallet-v3 = 0.25.0

I had a service-my-wallet-v3 on 24 version working. When the service was updated to 0.25.0 the feature to create new account into a wallet stoped and a i get this error:  Main wallet password incorrect.  I didn't change the password.

**Comment by jtormey** [comment]
Could you try completely uninstalling and reinstalling the service?

```sh
npm uninstall -g blockchain-wallet-service
npm install -g blockchain-wallet-service
```

**Comment by haardikk21** [comment]
Having the same issue. Very annoying.

Also, I see that the address_balance endpoint has been deprecated and the HD api just has a /balance endpoint - is it supposed to return the combined balance of all addresses in the account?

**Comment by jtormey** [comment]
That's correct, the [balance endpoint](https://github.com/blockchain/service-my-wallet-v3#check-hd-account-balance) will retrieve the balance for the xpub, which is the sum of all the address balances.

@haardikk21 do you only have this issue with the create endpoint? Could you start this service with `LOGLEVEL=debug`, and let me know if that gives more information? What happens if you completely uninstall and reinstall the service? Thanks.

**Comment by brunosalmeida** [comment]
@jtormey uninstall the package worked.

**Comment by yaswanth-rails** [comment]
Me too getting same issue even re installing the service. Please check below logs.

1495877207083 - debug: Logging in
1495877217934 - debug: Fetching wallet history
1495877609528 - debug: Recording smwv3 event name=recv
1495877642383 - error: ERR_PASSWORD

**Comment by aaressinfomedia** [comment]
Hello Yashwant-rails,

You can check your email for authorize IP address and click on it. that should help

**Comment by bobozar** [comment]
same isssue here. done all what is listed above can't get it to authenticate and my password is correct.

**Comment by rakeshrocket** [comment]
1541243815626 - debug: Logging in
1541243815926 - debug: Created new session token
1541243816427 - debug: Fetched wallet
1541243816440 - error: ERR_PASSWORD
